atlas_densities/densities/fitting.py

```diff
@@ -587,7 +614,7 @@ def linear_fitting(  # pylint: disable=too-many-arguments
     _check_average_densities_sanity(average_densities)
     _check_homogenous_regions_sanity(homogenous_regions)
 
-    hierarchy_info = get_hierarchy_info(region_map, root=region_name)
+    hierarchy_info = utils.get_hierarchy_info(region_map, root=region_name)
```

Comment:
I usually prefer importing the name rather then the dot notation, but that is minor

Reply:
When there are multiple things from a module, it gives more context when it includes the name, imo.  When things are all over the place, bare names don't give as much clarity, imo.